cluster/src/table_manager.rs

```diff
@@ -148,6 +133,30 @@ impl TableManager {
         }
     }
 
+    // todo: should accept schema id as param instead of schema name?
+    pub fn tables_by_schema(&self, catalog: &str, schema: &str) -> Vec<TableRef> {
```

Comment:
`tables_by_catalog_and_schema` seems too verbose, and `by_schema` denotes implicitly that the catalog should be provided because no schema can exists outside the catalog.

Reply:
I see `SpaceAndTable` in ceresdb and `RecordAndOffset` in rskafka. I think such naming is not so rare actually.

Reply:
Ok, but the code is outdated, and in the new codes, there is no such hard naming issue. :stuck_out_tongue_winking_eye: